packages/orbit-components/src/primitives/ButtonPrimitive/index.tsx

```diff
@@ -160,6 +160,7 @@ const ButtonPrimitive = React.forwardRef<HTMLButtonElement | HTMLAnchorElement,
         className={cx(
           className,
           "orbit-button-primitive font-base duration-fast group relative max-w-full items-center border-none text-center leading-none transition-all [&>*]:align-middle [&_.orbit-loading-spinner]:stroke-[currentColor]",
+          "flex shrink-0 grow-0 basis-auto",
```

Comment:
The `flex` can't be there. It's already defined below (line 172).
The other three classes can be simplified into `flex-none`.
So it's just
```suggestion
          "flex-none",
```